php/php-tarantool/php-tarantool-dev.spec

```diff
@@ -99,18 +93,7 @@ Package built for PHP %(%{__php} -r 'echo PHP_MAJOR_VERSION.".".PHP_MINOR_VERSIO
 
 %prep
 %setup -qc 
-mv %{github_name}-%{github_commit} NTS
-
-cd NTS
-: Bump version to avoid user confusion
-sed -i -e '/PHP_TARANTOOL_VERSION/s/0.1.0/%{version}-dev/' php_tarantool.h
-
-extver=$(sed -n '/#define PHP_TARANTOOL_VERSION/{s/.* "//;s/".*$//;p}' php_tarantool.h)
-if test "x${extver}" != "x%{version}%{?pre}%{?github_date:-dev}"; then
-   : Error: Upstream extension version is ${extver}, expecting %{version}%{?pre}%{?github_date:-dev}.
-   exit 1
-fi
-cd ..
+mv %{github_name}-%{github_version} NTS
```

Comment:
I always keep this control, as this is the most commonly encountered mistake in release process.